api/roles.rst

```diff
@@ -0,0 +1,751 @@
+Roles
+*****
+
+List
+====
+
+Required permission: ``admin.role``
+
+``GET`` -Request sent: ``/api/v1/roles/?full=true&_={id}``
+
+Response:
+
+.. code-block:: json
+   :force:
+
+   # HTTP-Code 200 OK
+
+   {
+      "record_ids": [
+         1,
+         2,
+         3
+      ],
+      "assets": {
```

Comment:
Lets make this more readable by not including `full=true` flag and `assets` content.

Reply:
Oh I guess it was not really correct at all in my first iteration ... Changed the request to the "standard" `/api/v1/roles` now so it should look better.